app/src/main/java/com/framgia/wsm/screen/main/MainViewModel.java

```diff
@@ -0,0 +1,165 @@
+                break;
+            case R.id.item_working_calendar:
+                setCurrentPageItem(Page.WORKING_CALENDAR, R.id.item_working_calendar);
+                break;
+            case R.id.item_holiday_calendar:
+                setCurrentPageItem(Page.HOLIDAY_CALENDAR, R.id.item_holiday_calendar);
+                break;
+            case R.id.item_statistic_of_personal:
+                setCurrentPageItem(Page.STATISTIC_OF_PERSONAL, R.id.item_statistic_of_personal);
+                break;
+            case R.id.item_overtime:
+                setCurrentPageItem(Page.OVERTIME, R.id.item_overtime);
+                break;
+            case R.id.item_come_late_leave_early:
+                setCurrentPageItem(Page.COME_LATE_LEAVE_EARLY, R.id.item_come_late_leave_early);
+                break;
+            case R.id.item_workspace_data:
+                setCurrentPageItem(Page.WORKSPACE_DATA, R.id.item_workspace_data);
+                break;
+            default:
+                break;
+        }
+        setStatusDrawerLayout(Constant.DRAWER_IS_CLOSE);
+        return true;
+    }
+
+    @Bindable
+    public String getStatusDrawerLayout() {
+        return mStatusDrawerLayout;
+    }
+
+    private void setStatusDrawerLayout(String statusDrawerLayout) {
+        mStatusDrawerLayout = statusDrawerLayout;
+        notifyPropertyChanged(BR.statusDrawerLayout);
+    }
+
+    public String getUsername() {
+        return mUser != null ? mUser.getName() : "";
+    }
+
+    public String getEmail() {
+        return mUser != null ? mUser.getEmail() : "";
+    }
+
+    @Override
+    public boolean onBackPressed() {
+        if (mStatusDrawerLayout.equals(Constant.DRAWER_IS_OPEN)) {
+            setStatusDrawerLayout(Constant.DRAWER_IS_CLOSE);
+            return true;
+        } else {
+            Fragment fragment = mViewPagerAdapter.getCurrentFragment();
+            if (fragment == mViewPagerAdapter.getFragment(Page.WORKING_CALENDAR)) {
+                MainContainerFragment containerFragment = (MainContainerFragment) fragment;
+                return containerFragment.onBackPressed();
+            } else {
+                if (fragment instanceof MainContainerFragment) {
+                    MainContainerFragment containerFragment = (MainContainerFragment) fragment;
+                    boolean isCanBack = containerFragment.onBackPressed();
+                    if (!isCanBack) {
+                        setCurrentPageItem(Page.WORKING_CALENDAR, R.id.item_working_calendar);
+                    }
+                    return true;
+                }
+            }
+        }
+        return false;
+    }
+
+    @IntDef({
+            Page.PERSONAL, Page.SETUP_PROFILE, Page.WORKING_CALENDAR, Page.HOLIDAY_CALENDAR,
+            Page.STATISTIC_OF_PERSONAL, Page.OVERTIME, Page.COME_LATE_LEAVE_EARLY,
+            Page.WORKSPACE_DATA
+    })
+    @interface Page {
+        int PERSONAL = 0;
```

Comment:
ERROR: 'member def type' have incorrect indentation level 8, expected level should be 4.